tasks/post-to-cmr/schemas/config.json

```diff
@@ -31,6 +31,11 @@
         "username": { "type": "string" },
         "password": { "type": "string" }
       }
+    },
+    "metaCheck": {
```

Comment:
If this property performs the check for a metadata file when its value is `false`, then it seems like it should be named `skipMetadataCheck`? Because you're only going to skip the metadata check when this property's value is `true`